### PR TITLE
update: Add setup.py

### PR DESCRIPTION
**Pull request by mrshu**
* Add the first version of setup.py in order to simplify creation of a
  real `wikiextractor` command.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

**Comment by mrshu** [comment]
@attardi Adding setup.py and then [uploading it to PyPI](https://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#uploading-your-project-to-pypi) will have at least the following andvantages:

1. It will be possible to install `wikiextractor` just by running `pip install wikiextractor` using standard Python distribution tool

2. The aforementioned command will also create a `wikiextractor` command in a crossplatform manner. Therefore it will not be necessary to locate  `WikiExtractor.py` in order to run it.

Please do let me know if I can be of any help here. Being able to install it in a normal 'Pythonic' way would make it way easier for other Python projects to make use of it.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+setup(
+    name='wikiextractor',
+
+    description='A script that extracts and cleans text from a Wikipedia'
+                'database dump',
+    author='Giuseppe Attardi',
```
Comment (mrshu) [inline setup.py]: @attardi I am not sure if you are the author but if you are not you might change this keyword and you might move yourself to `maintainer` and `maintainer_email`.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    author_email='attardi@di.unipi.it',
+    version='2.42',
+
+    url='https://github.com/attardi/wikiextractor',
+
+    license="GPL 3.0",
+    keywords=['text', 'nlp'],
```
Comment (mrshu) [inline setup.py]: @attardi I am not sure which keywords to use so please feel free to update this.

**Comment by mrshu** [comment]
@attardi Thank you for merging this PR.

I was wondering if it would be possible to upload this project to the Python Package Index (PyPI)? The instructions on how to do that can be found [here](https://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#uploading-your-project-to-pypi) and seem to be pretty straightforward.

Please do let me know if I can be of any help in that regard. Having this project on PyPI would help many other projects that use `WikiExtractor` (they could just list it in their `setup.py` or `requirements.txt` as a dependency) and also simplify its installation (one can just do `pip install wikiextractor` instead of having to download the file from this repository and putting it somewhere) and updating (one does not have to download a new version of the script from this repository but only needs to run `pip install --upgrade wikiextractor`).

**Comment by attardi** [comment]
Please go ahead, if you want.
I just wonder how to maintain new releases, ensuring that both repositories remain aligned.

**Pull request merged**